ptd_client_server/server.py

```diff
@@ -241,6 +250,38 @@ def cmd(self, cmd: str) -> Optional[str]:
         logging.info(f"Reply from ptd: {reply!r}")
         return reply
 
+    def _get_initial_range(self) -> None:
+        # Normal Response: ?Ranges,{Amp Autorange},{Amp Range},{Volt Autorange},{Volt Range}\r\n?
+        # Values: for autorange settings, -1 indicates ?unknown?, 0 = disabled, 1 = enabled
+        # For range values, -1.0 indicates ?unknown?, >0 indicates actual value
+        response = self.cmd("RR")
+        if response is None or response == "":
+            logging.error("Can not get initial range")
+            exit(1)
+
+        response_list = response.split(",")
+
+        def get_range_from_ranges_list(param_num: int) -> str:
+            try:
+                if (
+                    response_list[param_num] == "0"
+                    and float(response_list[param_num + 1]) > 0
+                ):
+                    return response_list[param_num + 1]
+            except ValueError:
+                logging.info(
+                    "Can not get ptd range value for "
+                    + ("Amps" if param_num == 1 else "Volts")
+                )
```

Comment:
I think `logging.warning` is more appropriate here. Also, please pass `"Amps"`/`"Volts"` as an argument.

```suggestion
                logging.warning(f"Can not get ptd range value for {setting}")
```